### PR TITLE
feat: Wish List page added

### PR DESCRIPTION
**Pull request by Manishsharma203**
For preview check link below:
`https://7xyp23.csb.app/pages/wishList.html`

### CODE REVIEW

**Review by masraniJay — APPROVED** [review]
Looks good.

File: index.html [hunk 1]
```diff
@@ -1,11 +1,61 @@
...
+      <div class="navbar bg-primary p-2" style="z-index: 100">
+        <input id="menu-icon" type="checkbox" />
+        <label class="menu-label" for="menu-icon"></label>
+        <div class="nav-brand">
+          <div class="h3 text-bold color-primary box-3d p-1-x bg-secondary">
+            CALYX Mart
+          </div>
+        </div>
+        <div class="nav-content">
+          <!-- <h3>Navbar</h3>
```
Comment (ooreally) [inline index.html:24]: Remove commented code

File: pages/homePage.html [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+      <div class="navbar bg-primary p-2" style="z-index: 100">
+        <input id="menu-icon" type="checkbox" />
+        <label class="menu-label" for="menu-icon"></label>
+        <div class="nav-brand">
+          <div class="h3 text-bold color-primary box-3d p-1-x bg-secondary">
+            CALYX Mart
+          </div>
+        </div>
+        <div class="nav-content">
+          <!-- <h3>Navbar</h3>
```
Comment (ooreally) [inline pages/homePage.html:26]: remove commented code

**Review by ooreally — CHANGES_REQUESTED** [review]
Looks good to merge.
Suggestion: remove commented code

File: index.html (hunk 1 above)
Comment (Manishsharma203) [reply index.html:24]: sure

**Pull request merged**